WTWSMS/common/religions/00_religions.txt

```diff
@@ -4775,6 +4801,19 @@ bosporan_group = {
 		can_grant_divorce = yes
 		can_call_crusade = yes
 		can_grant_invasion_cb = invasion
+
+		short_reign_opinion_year_mult = 0 # No short reign penalty
+
+		character_modifier = { 
+			fertility = 0.10
+			learning = 2
+			ai_honor = 1
+		}
+	
+		defensive_attrition = yes
+		independence_war_score_bonus = 50
+		attacking_same_religion_piety_loss = yes
+		has_heir_designation = yes
```

Comment:
That still doesn't explain the heir designation.

Then I will just leave the Hypsistarians as they are and ask for more research to be made. It is better that we try to make things clear and look at what features they could get.